.github/workflows/deploy-edge-app.yml

```diff
@@ -0,0 +1,44 @@
+---
+# Manually deploy any edge app to either the staging (QC) or production environment.
+# Always uses the latest master branch.
+name: Deploy Edge App
+
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: 'Environment to deploy to (stage or production)'
+        required: true
+        default: 'stage'
+        type: choice
+        options:
+          - stage
+          - production
+
+      edge_app_name:
+        description: 'Folder name of the edge app (under edge-apps/)'
+        required: true
+        default: ''
```

Comment:
That's inconvenient. Make it a selector, with a list of apps.

Reply:
@renatgalimov We can make a list of apps available in the selector by 

```
options:
   - fake-dashboard
   - 3d-text
   - google-calendar
```

but **hardcoding** the list of apps in the workflow file would **require manual updates every time a new edge app is added,** which is not maintainable. Other option, we can run sub action to fetch the app inside the **edge-app dir** and list the avilavle edge-apps names and use it here.

Reply:
@nicomiguelino What do you think? 

I see your comments about this, https://github.com/Screenly/Playground/pull/264#discussion_r2096056012 - but is it better to enter the name manually?